cmds/ps/ps.go

```diff
@@ -49,6 +50,11 @@ func init() {
 	flag.BoolVar(&flags.all, "e", false, "Select all processes.  Identical to -A.")
 	flag.BoolVar(&flags.x, "x", false, "BSD-Like style, with STAT Column and long CommandLine")
 	flag.BoolVar(&flags.nSidTty, "a", false, "Print all process except whose are session leaders or unlinked with terminal")
+
+	if len(os.Args) > 1 {
+		flags.all = true
+		flags.aux = isPermutation(os.Args[1], "aux")
```

Comment:
That is true. We chose this implementation because we thought the value from "aux" was the most valuable to the user. If necessary, I could separate the options. What do you think?